draft-ietf-anima-brski-prm.md

```diff
@@ -1386,17 +1433,17 @@ In addition, the following processing SHALL be performed for PVR contained in RV
 
 * agent-signed-data: The MASA MAY verify this data to issue "agent-proximity" assertion.
   If so, the agent-signed-data MUST contain the pledge product-serial-number, contained in the "serial-number" field of the PVR (from "prior-signed-voucher-request" field) and also in "serial-number" field of the RVR.
```

Comment:
Is there intentionally no check against the X520SerialNumber field of the pledge IDevID certificate?

Reply:
It is actually intended to follow the same path as in [BRSKI](https://datatracker.ietf.org/doc/html/rfc8995#name-masa-verification-of-the-pl).  we stated this in section 7.3.2 as 

serial-number: product-serial-number of pledge. The registrar MUST verify that the X520SerialNumber field of the pledge IDevID certificate matches the serial-number value in the PVR. In addition, it MUST be equal to the serial-number value contained in the agent-signed data of PVR.

Reply:
Yes, this text is on the check by the **Domain Registrar**, where IDevID X520SerialNumber is checked.
For the check by the MASA, however, you omitted the IDevID X520SerialNumber check.

I would have expected the MASA does the very same checks as the Registrar plus the comparison with the RVR field.

Reply:
Good catch, the MASA should do the check as well, that was unintentional. We intended to not change the general MASA duties for the checks. So essentially, they should be done as in plain-vanilla BRSKI as described in [Section 5.5.5 of RFC 8995 ](https://datatracker.ietf.org/doc/html/rfc8995#name-masa-verification-of-the-pl). We just wnted to emphasize the extra steps for the MASA through the new assertion type.

Reply:
Clarified from my understanding and can be closed if incorporated.